models/keep3r_network/ethereum/keep3r_network_ethereum_view_job_credits_log.sql

```diff
@@ -0,0 +1,123 @@
+{{ config (
+    materialized = 'view',
```

Comment:
we also shouldn't need ```materialized``` property in all these models, as view is the default based on your additions to dbt_project yaml file